gcc/config/arc/arc.opt

```diff
@@ -102,6 +102,25 @@ Same as -mcpu=archs
 ; END backward-compatibility aliases, translated by DRIVER_SELF_SPECS
 
 TargetVariable
+unsigned int arc_jli_func_addr = 0
```

Comment:
Then when is it necessary? I see other options declare variables, hence the confusion.